{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:redhat:ansible:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "1.2.2",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2013-4259"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "ansible",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "\u003c=",
                        "version_value": "1.2.2"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "redhat"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "runner/connection_plugins/ssh.py in Ansible before 1.2.3, when using ControlPersist, allows local users to redirect a ssh session via a symlink attack on a socket file with a predictable name in /tmp/."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-264"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "http://www.ansible.com/security",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "http://www.ansible.com/security"
        },
        {
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=998223",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch"
          ],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=998223"
        },
        {
          "name": "https://groups.google.com/forum/#!topic/ansible-project/UVDYW0HGcNg",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://groups.google.com/forum/#!topic/ansible-project/UVDYW0HGcNg"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "MEDIUM",
        "accessVector": "LOCAL",
        "authentication": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 1.9,
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "vectorString": "AV:L/AC:M/Au:N/C:P/I:N/A:N",
        "version": "2.0"
      },
      "exploitabilityScore": 3.4,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "LOW",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2018-10-30T16:28Z",
  "publishedDate": "2013-09-16T19:14Z"
}